### PR TITLE
[ IDE ] refine command

### PR DESCRIPTION
**Pull request by gallais**

### CODE REVIEW

File: src/Core/TT/Views.idr [hunk 1]
```diff
@@ -3,7 +3,7 @@ module Core.TT.Views
 import Core.Env
 import Core.TT
 
-import Data.SnocList
+import Data.SnocList -- untile 0.6.0 release
```
Comment (buzden) [inline src/Core/TT/Views.idr:6]: "until"?

Comment (gallais) [reply src/Core/TT/Views.idr:6]: bootstrapping problems

Comment (buzden) [reply src/Core/TT/Views.idr:6]: > bootstrapping problems

Yeah, I mean the word spelling

Comment (gallais) [reply src/Core/TT/Views.idr:6]: :smile:

**Comment by ohad** [comment]
Not a review, but please update the [docs](https://github.com/gallais/Idris2/blob/refine-hole/docs/source/implementation/ide-protocol.rst) too please.

**Comment by michaelmesser** [comment]
```
test : Nat -> Nat
test = ?h0
```

`:refine 2 h0 (+)` outputs `+ ?h0_0` but it should be `(+ ?h0_0)` or `(+) ?h0_0`

**Pull request merged**